### PR TITLE
Edit: The Spanish texts in the Resources folder have been updated.

### PR DESCRIPTION
**Pull request by M1atias**
Hello i'm Matias and this is my colaborations to this proect:
I've working in a updated of the Spanish texts in the Resources folder. There were some texts that were still in English so I decided to translate them, there are still some translations pending if my collaboration is accepted I will continue working on those translations. If you have any suggestions I am at your disposal to listen to them

### CODE REVIEW

File: PKHeX.Core/Resources/text/script/const_pt_es.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 0048	Inicial	0:Sin Elegir,387:Turtwig,390:Chimchar,393:Piplup
-0122	Togepi Egg	0:Visited Statue,1:Received HM01,2:Cynthia Disappeared,3:Cynthia Gives Egg,4:Egg Pending,5:Egg Received
+0122	Togepi Egg	0:Visited Statue,1:Received HM01,2:Cynthia Desaparecio,3:Cynthia da un huevo,4:Huevo Pendiente,5:Huevo Recibido
```
Comment (Lusamine) [inline PKHeX.Core/Resources/text/script/const_pt_es.txt:2]: Thanks for your contribution!  In this line, should "Visited Statue" and "Received HM01" also be translated?

Comment (M1atias) [reply PKHeX.Core/Resources/text/script/const_pt_es.txt:2]: You are welcome ! Yes as I said above in the commentary there are still some untranslated texts in English, if my collaboration seems ok and is accepted I will continue with the translation of the missing texts.

Comment (Lusamine) [reply PKHeX.Core/Resources/text/script/const_pt_es.txt:2]: Ah, I was wondering why only part of that line was translated.  Go ahead and do the others you were planning.  You may want to keep an eye on the capitalization to stay consistent ("Huevo Recibido" vs "Huevo recibido" and "No activado" vs "No Activado").

**Comment by M1atias** [comment]
Update the translation you mentioned and perform the Spanish translation of new files.

**Pull request merged**